dist/PriorityQueue.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
```

Comment:
Add `dist` to your `.gitignore` so this transpiled code doesn't appear in your repo.